testnets/wavehashtestnet/assetslist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "wavehashtestnet",
+    "assets": [
+      {
+        "description": "WaveHash A Permissioned Blockchain",
+        "denom_units": [
+          {
+            "denom": "uwahax",
+            "exponent": 0
+          },
+          {
+            "denom": "wahax",
+            "exponent": 6
+          }
+        ],
+        "base": "uwaha",
+        "name": "WaveHash",
+        "display": "wavehash",
```

Comment:
display value must be one of the defined denom_units, so either uwahax or wahax (typically will be the larger)